src/napari_imagej/widgets/menu.py

```diff
@@ -255,6 +266,45 @@ def disable_popup(self):
         )
 
 
+class REPLButton(IJMenuButton):
+    def __init__(self, viewer: Viewer):
+        super().__init__(viewer)
+        self.viewer = viewer
+
+        self.setEnabled(False)
+
+        icon = QColoredSVGIcon(resource_path("repl"))
+        self.setIcon(icon.colored(theme=viewer.theme))
```

Comment:
If you could replace this with the way that we set the icon [here](https://github.com/imagej/napari-imagej/blob/75cedb62a87a8d8ec20a1669133c517bf48c0b6e/src/napari_imagej/widgets/menu.py#L125), then the icon will be recolored when the user changes the theme.

Reply:
Fixed with [60f1f77](https://github.com/imagej/napari-imagej/pull/265/commits/60f1f77b89ada2431b3f90e0409e9d3f185f4bfe)